### PR TITLE
Added null object constants.

### PR DESCRIPTION
**Pull request by vishwesh3**
fix:#819

### CODE REVIEW

File: src/account/accountReducers.js [hunk 1]
```diff
@@ -58,7 +59,7 @@ export default (state: AccountState = initialState, action: Action) => {
 
       const { type, ...newAccount } = action; // eslint-disable-line no-unused-vars
 
-      if (accountIndex === -1) {
+      if (accountIndex === NULL_ACCOUNT_INDEX) {
```
Comment (borisyankov) [inline src/account/accountReducers.js]: I don't think this is a good idea. -1 is a well known 'not known' value, while we add nothing by adding a constant here.

File: src/nullObjects.js [hunk 2]
```diff
@@ -1,4 +1,11 @@
-import type { Account, User } from './types';
+/* @flow */
+import type {
+  Account,
+  User,
+  Stream,
+  Message,
+  Subscription,
+} from './types';
```
Comment (borisyankov) [inline src/nullObjects.js]: This probably fits in a single line?

File: src/nullObjects.js [hunk 3]
```diff
@@ -12,3 +19,48 @@ export const NULL_USER: User = {
   avatarUrl: '',
   email: '',
 };
+
+export const NULL_ACCOUNT_INDEX: number = -1;
+export const NULL_USER_INDEX: number = -1;
+export const NULL_MESSAGE_INDEX: number = -1;
+
+export const NULL_STREAM: Stream = {
+  stream_id: -1,
```
Comment (borisyankov) [inline src/nullObjects.js]: Maybe we want to use 0 for the null numbers.

File: src/account/accountReducers.js (hunk 1 above)
Comment (vishwesh3) [inline src/account/accountReducers.js]: Done 👍

File: src/nullObjects.js (hunk 2 above)
Comment (vishwesh3) [inline src/nullObjects.js]: Done 👍

File: src/nullObjects.js (hunk 3 above)
Comment (vishwesh3) [inline src/nullObjects.js]: Done 👍

**Comment by vishwesh3** [comment]
@borisyankov Updated 👍

**Comment by borisyankov** [comment]
Looks good 👍

**Pull request merged**